src/System.Private.Uri/src/System/UriFormatException.cs

```diff
@@ -2,12 +2,22 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#if netstandard10
```

Comment:
The only dependencies this assembly has is coreclr, which is available for netstandard1.0. I guess we *could* rev the NuGet moniker to 1.3 but I don't really see the point of doing it since this can be packaged as 1.0.